pypeit/tests/test_fluxspec.py

```diff
@@ -67,9 +67,9 @@ def test_run_from_spec1d(kast_blue_files, master_dir):
     # Write
     FxSpec.write_science(data_path('tmp.fits'))
     # Master
-    FxSpec.save_master()
+    FxSpec.save_master(FxSpec.sens_dict)
```

Comment:
Seems a bit odd to have to pass this in as it is
obviously in the Object already.  But ok.

Reply:
I think argument lists make it more clear what a functions are doing when you have objects as complicated as we do. The best example is the General class in arclines which is a giant piece of spaghetti code where variables get toggled implicitly in many places and the result is you have no idea what the code is actually changing or doing, and all the dependencies are obscured. For this reason I structured the new land and save master algorithms which can be overloaded to have an argument.